code/game/machinery/autolathe.dm

```diff
@@ -490,3 +491,14 @@
 /obj/machinery/autolathe/proc/check_disabled_callback()
 	if(!wires.is_cut(WIRE_AUTOLATHE_DISABLE))
 		disabled = FALSE
+
+/obj/machinery/autolathe/syndicate
+	name = "suspicious autolathe"
+	board_type = /obj/item/circuitboard/autolathe/syndi
+
+/obj/machinery/autolathe/syndicate/New()
+	..()
+	if(files)
+		qdel(files)
```

Comment:
```suggestion
		QDEL_NULL(files)
```
This may work better here as its probably best to null the entire thing out if youre about to re-assign it